### ISSUE TITLE
manage_service_user is unused

### ISSUE THREAD

**Issue by jasperla** [mwhahaha/puppet-pgpool#4]
Hi,

It seems the `manage_service_user` parameter is not wired to anything and thus not very useful as is.

**Comment by mwhahaha** [comment]
You are correct. I believe this is an artifact of my initial writing of the module.  Some pgpool packages just assume the postgres user is there and it's not configureable. This flag was intended to allow for the creation of the service user if you were installing pgpool on a box that didn't have postgres  installed.  It can either be removed or the functionality can be added in.
